ampersand-select-view.js

```diff
@@ -173,8 +179,20 @@ SelectView.prototype.remove = function () {
     this.el.removeEventListener('change', this.onChange, false);
 };
 
+SelectView.prototype.reset = function() {
+    if(this.startingValue) {
+        this.setValue(this.startingValue, true);
+    } else {
+        this.clear();
+    }
+};
+
 SelectView.prototype.clear = function() {
-    this.setValue('', true);
+    if(this.unselectedText) {
+        this.setValue(undefined, true);
+    } else {
+        this.setValue(this.select.options[0].value, true);
```

Comment:
Would something like this handle it?

    if(this.unselectedText || !this.select.options[0].value) {
      this.setValue(undefined, true);
    } else {
      this.setValue(this.select.options[0].value, true);
   }